other/materials_designer/Introduction.ipynb

```diff
@@ -14,13 +14,12 @@
     "\n",
```

Comment:
<p>We should organize this in sections:</p><p> </p><ol><li>Builders / Transformations</li><li class="ql-indent-1">1.1. Interface Creation</li><li class="ql-indent-1">1.2. Etc</li><li>Import Data</li><li class="ql-indent-1">Import materials from JARVIS</li><li class="ql-indent-1">Import materials from File in ASE-supported formats</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>